tools/toolchains/arm.py

```diff
@@ -556,7 +564,9 @@ def __init__(self, target, *args, **kwargs):
                     "ARM/ARMC6 compiler support is required for ARMC6 build"
                 )
 
-        if getattr(target, "default_toolchain", "ARMC6") == "uARM":
+        self.check_c_lib_supported(target, "arm")
+
+        if getattr(target, "default_toolchain", "ARMC6") == "uARM" or getattr(target, "default_lib", "std") == "small":
```

Comment:
```suggestion
        if (
            getattr(target, "default_toolchain", "ARMC6") == "uARM"
            or getattr(target, "default_lib", "std") == "small"
        ):
```
?

Reply:
One of your earlier reviews comments to use without brackets like below
 if getattr(target, "default_toolchain", "ARM") == "uARM"
  or getattr(target, "default_lib", "std") == "small":

Reply:
https://github.com/ARMmbed/mbed-os/pull/12068/files/d461197d5c183fdf506e41405bf6439be96bd10c#r357169368